lib/consul/migrate/cli.rb

```diff
@@ -0,0 +1,71 @@
+require 'consul/migrate/version'
+require 'consul/migrate/client'
+require 'consul/migrate/defaults'
+require 'thor'
+require 'yaml'
+require 'fileutils'
+
+module Consul
+  module Migrate
+    class Cli < Thor
+
+      attr_accessor :client
+
+      def initialize(*args)
+        super
+        read_config
+      end
+
+      desc 'version', 'Display consul-migrate version'
+      def version
+        say "#{VERSION}"
```

Comment:
Ha, that's what the bundle gem scaffolding generated, but I side with you on it being unnecessary.